src/metal.cs

```diff
@@ -304,6 +348,21 @@ public partial interface MTLComputeCommandEncoder : MTLCommandEncoder {
 		[Abstract]
 		[Export ("setBytes:length:atIndex:")]
 		void SetBytes (IntPtr bytes, nuint length, nuint index);
+
+		[iOS (10,0), TV (10,0), NoWatch, Mac (10,12)]
+		[Abstract]
+		[Export ("setStageInRegion:")]
+		void SetStageInRegion (MTLRegion region);
+
+		[iOS (10,0), TV (10,0), NoWatch, NoMac]
+		[Abstract]
+		[Export ("updateFence:")]
+		void UpdateFence (IMTLFence fence);
```

Comment:
In a sense, the param does give enough info, but Update in Metal might mean A LOT of things. SO I have a strange feeling that might confuse people. But I'll remove it.